docs/manage/CLI.md

```diff
@@ -99,23 +99,61 @@ To later update the Viam CLI tool, you can use the steps above to reinstall the
 
 ## Authenticate
 
-Once you have successfully installed the Viam CLI, you need to authenticate your device for CLI usage with your Viam app account before you can control your robots with the CLI.
-Do this by issuing the command:
+Once you have [installed the Viam CLI](#install), you must authenticate your CLI session with Viam in order to run CLI commands.
 
-```sh {class="command-line" data-prompt="$"}
-viam login
-```
+You can authenticate your CLI session using either a personal access token or your organization's API key.
+To use your organization API key to authenticate, you must first [create an organization API key](#create-an-organization-api-key).
+
+* To authenticate your CLI session using a personal access token:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam login
+   ```
+
+   This will open a new browser window with a prompt to start the authentication process.
+   If a browser window does not open, the CLI will present a URL for you to manually open in your browser.
+   Follow the instructions to complete the authentication process.
 
-This will open a new browser window with a prompt to start the authentication process.
-If a browser window does not open, the CLI will present a URL for you to manually open in your browser.
-Follow the instructions to complete the authentication process.
+* To authenticate your CLI session using your organization API key:
+
+   ```sh {class="command-line" data-prompt="$"}
+   viam login api-key --key-id <api-key-uuid> --key <api-key-name>
+   ```
+
+   If you haven't already, [create an organization API key](#create-an-organization-api-key) to use this authentication method.
 
-{{% alert title="Info" color="info" %}}
 An authenticated session is valid for 24 hours, unless you explicitly [log out](#logout).
 
 After the session expires or you log out, you must re-authenticate to use the CLI again.
```

Comment:
Question: Do lines 125-127 also apply to sessions authenticated via org API keys? Assuming: yes.

Reply:
Org api keys do not expire, so kind of. Probably fine to leave as is.